test/test_type.jl

```diff
@@ -673,16 +673,16 @@ isdir(outdir) && rm(outdir, recursive = true)
                       RealT
             end
 
-            @test eltype(@inferred max_abs_speed_naive(u_ll, u_rr, orientation, equations)) ==
+            @test typeof(@inferred max_abs_speed_naive(u_ll, u_rr, orientation, equations)) ==
```

Comment:
You forgot this. Sometimes I may not go back to the files that you have already reviewed to review again myself. Please make the request cover all changes in one file. Thanks!